### PR TITLE
Update repo

### PR DESCRIPTION
**Pull request by srisankethu**
Things I have done in this PR:
* Slightly modified README.md
* Add a list of Ethnic Slurs(ethnic_slurs.md)
* Add a list for Gender Bias. This list seems too exhaustive right now. Just consider rewriting all the relevant words ending with "-man"(includes woman) to be gender neutral. [Here](https://www.thefreedictionary.com/words-that-end-in-man) is a list of words ending with "man" . I think if we can detect "man" or "woman" and "men" or "women" in a word and replace it with "person" and "people" respectively, it would be easier. What do you think?
* Add a list for Disability Bias
* Add templates for remaining bias(empty .md files)

### CODE REVIEW

**Comment by srisankethu** [comment]
@pdlla I am confused to what "Cultural bias" and "Sexual Orientation bias" files should contain? I felt "Sexual Orientation bias" can be covered in "Gender bias". So should "Sexual Orientation bias" contain list of slurs against LGBTQ? Can you clear me on this?

File: ethnic_slurs.md [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+Gyppo: A Romani person, Egyptians  
+gippo: A Romani person, Egyptians  
+gypo: A Romani person, Egyptians  
+gyppie: A Romani person, Egyptians  
+gyppy: A Romani person, Egyptians  
+gipp: A Romani person, Egyptians  
+Hairyback: Afrikaners  
+Half-breed: A person of mixed race  
+Haole: a non-Hawaiian whites  
+Heeb: a Jew
```
Comment (minimapletinytools) [inline ethnic_slurs.md]: "Jew" is sometimes considered offensive. These should be translated to "a Jewish person"  https://en.wiktionary.org/wiki/Jew

File: ethnic_slurs.md [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+Hick: an uneducated or unsophisticated person from the countryside  
+Hike: an Italian immigrant  
+Hillbilly: an American of Appalachian or Ozark heritage  
+Honky: a White person  
+honkey: a White person  
+honkie: a White person  
+Hori: a Māori  
+Hun: Germans, a Protestant in Northern Ireland  
+Hunky: A central Central European laborer  
+Hymie: a Jew
```
Comment (minimapletinytools) [inline ethnic_slurs.md]: same comment

File: disability_bias.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+able-bodied: without disabilities
```
Comment (minimapletinytools) [inline disability_bias.md]: Many of these aren't correct replacements (wrong form) or aren't offensive terminology to begin with (e.g. sight-impaired). Can we just delete this file all together. Or update based on https://en.wikipedia.org/wiki/List_of_disability-related_terms_with_negative_connotations

File: ethnic_slurs.md [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+Abie: a Jewish man  
+Abo: australian aboriginal person  
+Abbo: australian aboriginal person  
+Afro engineering: shoddy, makeshift or unconventional, makeshift workmanship  
+African engineering: shoddy, makeshift or unconventional, makeshift workmanship  
+Ali Baba: an Iraqi suspected of criminal activity  
+Alligator bait: a black child  
+Gator bait: a black child  
+Annamite: a Vietnamese person  
+Ape: A black person
```
Comment (minimapletinytools) [inline ethnic_slurs.md]: I think some of these need to be removed. Ape is used in inoffensive ways often enough that this suggestion isn't useful out of context. Having your spell checker offering "a black person" as a replacement for "Ape" is offensive in its own way as well.

File: gender_bias.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+actress: actor
```
Comment (minimapletinytools) [inline gender_bias.md:1]: Lets delete this one for now. I understand this file is to replace all gendered terms with gender neutral versions but I think that's maybe not the context in which this dictionary will be used.

File: ethnic_slurs.md (hunk 4 above)
Comment (srisankethu) [reply ethnic_slurs.md]: I thought I removed general words used as a slur. Thanks for pointing out.

File: gender_bias.md (hunk 5 above)
Comment (srisankethu) [reply gender_bias.md:1]: You mean I should delete this file? I don't see whats wrong in suggesting gender-neutral words instead?

File: ethnic_slurs.md (hunk 2 above)
Comment (srisankethu) [reply ethnic_slurs.md]: Updated it

File: ethnic_slurs.md (hunk 1 above)
Comment (srisankethu) [reply ethnic_slurs.md]: Updated it

File: disability_bias.md (hunk 3 above)
Comment (srisankethu) [reply disability_bias.md]: Thanks for the link. I have written some of these from [a Microsoft guide](https://docs.microsoft.com/en-us/style-guide/bias-free-communication).

File: gender_bias.md (hunk 5 above)
Comment (minimapletinytools) [reply gender_bias.md:1]: I see, in the context of code style guide this makes more sense. Ok, lets keep the word then.

File: ethnic_slurs.md (hunk 4 above)
Comment (srisankethu) [reply ethnic_slurs.md]: I have removed such similar words

**Review by minimapletinytools — APPROVED** [review]
Cool. Go ahead and merge when you are done with all changes. If you'd like to do more work just open up a new PR. Thanks for your help! I'll pay out the bounty as we discussed later today or tomorrow.

File: disability_bias.md (hunk 3 above)
Comment (srisankethu) [reply disability_bias.md]: Updated it

**Pull request merged**